pkg/workload/tpcc/tpcc.go

```diff
@@ -300,6 +303,11 @@ var tpccMeta = workload.Meta{
 		g.flags.StringVar(&g.queryTraceFile, `query-trace-file`, ``, `File to write the query traces to. Defaults to no output`)
 		// Support executing a query file before each transaction.
 		g.flags.StringVar(&g.txnPreambleFile, "txn-preamble-file", "", "queries that will be injected before each txn")
+		g.flags.StringVar(&g.asOfSystemTime, "as-of", "",
+			"Timestamp at which the query should be run."+
```

Comment:
Which query? The last sentence should probably be the first, otherwise this is ambiguous. We should explicitly say that this is an optional parameter to specify AOST; used exclusively in conjunction with the TPCC consistency checks. Technically, it should be a sub-parameter of `expensive-checks`.

Reply:
Thank you for the feedback. Made the changes as suggested.